### PR TITLE
feat: add landle and test button #63

### PR DESCRIPTION
**Pull request by flaviafreirec**
Issue: feat #63

<details open> 
  <summary>
    <b>Feature</b>

  </summary>

</details>

- **Description**
This Pull Request implement the UI Component Manager `landle` for develop and test components and add a test button to the `Testinho.tsx` file to facilitate testing and integration with Landle.

- **Cause**
Problems with component tests when they were done directly in the file `App.tsx`

- **Solution**
We'll use `landle` to enhance tests of the components of our project.
</details>

<details open> 
  <summary>
    <b>Changelog</b>

- add `landle` dependencies on `package.json`
- add a `test button` on `Testinho.tsx`
- add new file `Testinho.stories.tsx` for tests on `landle`
  </summary>

</details>

<details open> 
  <summary>
    <b>Visual evidences :framed_picture:</b>

testttt

  </summary>

</details>

<details open> 
  <summary>
    <b>Checklist</b>
  </summary>

  - [x] Issue linked
  - [x] Build working correctly
  - [x] Tests created
</details>

### CODE REVIEW

File: src/components/Testinho/Testinho.stories.tsx [hunk 1]
```diff
@@ -0,0 +1,5 @@
+import type { Story } from '@ladle/react';
+
+import Testinho from './Testinho';
+
+export const World: Story = () => <Testinho />;
```
Comment (aalmeida00) [inline src/components/Testinho/Testinho.stories.tsx:5]: Aqui precisa ser o nome do componente

```suggestion
export const Testinho: Story = () => <Testinho />;
```

File: src/components/Testinho/Testinho.tsx [hunk 2]
```diff
@@ -1,7 +1,27 @@
-import React from 'react';
+import React, { useState } from 'react';
 
 const Testinho = ({ name = 'World' }) => {
-  return <h1>Hello, {name}!</h1>;
+  const [clicked, setClicked] = useState(false);
+  const [newName, setNewName] = useState(name);
```
Comment (aalmeida00) [inline src/components/Testinho/Testinho.tsx:5]: @Alecell 

Pedi pra ela escrever esse teste aqui no testinho porque vou usar ele pra deixar de exemplo pro Zunstand no pr #58

**Review by aalmeida00 — CHANGES_REQUESTED** [review]

File: package.json [hunk 3]
```diff
@@ -4,6 +4,7 @@
   "private": true,
   "main": "build/electron/main.js",
   "dependencies": {
+    "@ladle/react": "^2.17.2",
```
Comment (Alecell) [inline package.json:7]: Esse tipo de dependencia precisa ir pro `devDependencies`, não é um `dependencie`

**Review by Alecell — CHANGES_REQUESTED** [review]

**Review by Alecell — APPROVED** [review]

**Comment by Alecell** [comment]
@aalmeida00 só falta vc aprovar man!

**Review by aalmeida00 — APPROVED** [review]

**Pull request merged**